.github/workflows/test-macos.yaml

```diff
@@ -41,4 +42,6 @@ jobs:
           rubygems: 3.3.26
           bundler: 2.3.26 # needed to fix issue with steep on Ruby 3.0/3.1
       - run: bundle install
+      - if: ${{ matrix.ruby == '3.4' }}
+        run: gem update --system 3.5.21 # the latest gem causes 3.4 macos failues
```

Comment:
Can you link to the issue that made us lock this (maybe it's in [this page](https://github.com/rubygems/rubygems/issues?q=is%3Aissue+mac+is%3Aclosed)), so we know when to unlock it?